anchorecli/clients/apiexternal.py

```diff
@@ -830,31 +831,34 @@ def get_subscription_types(config):
 
     return(ret)
 
-# repo clients
 
-def add_repo(config, input_repo, autosubscribe=False, lookuptag=None):
-    userId = config['user']
+def add_repo(config, input_repo, auto_subscribe=False, lookup_tag=None, dry_run=False):
+    user_id = config['user']
     password = config['pass']
     base_url = config['url']
 
-    ret = {}
+    url = os.path.join(base_url, 'repositories')
 
-    base_url = re.sub("/$", "", base_url)
-    url = '/'.join([base_url, "repositories?repository="+input_repo+"&autosubscribe="+str(autosubscribe)])
-    if lookuptag:
-        url = url + "&lookuptag="+str(lookuptag)
+    query = {
```

Comment:
perfect

Reply:
Url join is available that will be more correct than the os path. Not sure if it's available for 2.7. https://docs.python.org/3/library/urllib.parse.html